assembl/static2/js/app/graphql/VoteSession.graphql

```diff
@@ -24,13 +25,21 @@ fragment tokenVoteSpecification on TokenVoteSpecification {
   }
   voteSpecTemplateId
   voteType
+}
+
+fragment tokenVoteSpecificationResults on TokenVoteSpecification {
   myVotes {
     ... on TokenVote {
       voteValue
       proposalId
       tokenCategoryId
     }
   }
+  numVotes
+  tokenVotes {
+    tokenCategoryId
+    numToken
```

Comment:
isn't it more `tokenCount` ?

Reply:
(I'm not a big fan of `num` in variables name but it also works I guess: `numToken` => "number of tokens"

Reply:
That's right, number of tokens. I wanted some consistency on the naming. We already have numContributors, numPosts...